build.gradle

```diff
@@ -129,6 +129,12 @@ tasks.withType(JacocoReport) {
         csv.enabled false
         html.destination file("${buildDir}/jacocoHtml")
     }
+
+    afterEvaluate {
+        classDirectories.setFrom(files(classDirectories.files.collect {
+            fileTree(dir: it, exclude: ['bio/terra/model/**'])
```

Comment:
Good news, I noticed that the generated openapi classes all have the `@Generated` annotation and I found that a newer version of jacoco will automatically ignore classes with this annotation. We're using 0.8.2 and this was added in [0.8.3](https://github.com/jacoco/jacoco/releases/tag/v0.8.3)

The bad news is that the antlr generated code doesn't use `@Generated`, [yet](https://github.com/antlr/antlr4/issues/2568), so that will still need an exclusion rule.

Reply:
Oh interesting!  Lemme try to bump the library and remove the model package exclusion and add the grammar one